src/components/NcListItem/NcListItem.vue

```diff
@@ -782,9 +782,9 @@ export default {
 	position: relative;
 	flex: 0 0 auto;
 	justify-content: flex-start;
-	padding: 8px 10px;
+	padding: calc(var(--default-line-height) / 2) calc(var(--default-clickable-area) / 4);
 	// 4px padding for the focus-visible styles. Width is reduced to compensate it
-	margin: 4px;
+	margin: 2px;
```

Comment:
Comment makes no sense then anymore.
Also does this change the look on stable29?